OpenEdX/Data/AppStorage.swift

```diff
@@ -279,4 +292,5 @@ public class AppStorage: CoreStorage, ProfileStorage, WhatsNewStorage, CourseSto
     private let KEY_ALLOWED_DOWNLOAD_LARGE_FILE = "allowedDownloadLargeFile"
     private let KEY_RESET_APP_SUPPORT_DIRECTORY_USER_DATA = "resetAppSupportDirectoryUserData"
     private let KEY_LAST_USED_SOCIAL_AUTH = "lastUsedSocialAuth"
+    private let KEY_DISCUSSION_NOTIFICATIONS_SETTING_STATUS = "discussionnotificationsSettingStatus"
```

Comment:
Please update name from `discussionnotificationsSettingStatus` to `discussionNotificationsSettingStatus`